packages/js/src/core/fetch-token.test.ts

```diff
@@ -32,7 +30,7 @@ describe('fetch access token by providing valid refresh token', () => {
         refreshToken: 'refresh_token',
         scope: ['read', 'register', 'manage'],
       },
-      conditional(isNode() && fetchFunction)
+      fetchFunction // Always passing `fetchFunction` since Jest has no `fetch()`
```

Comment:
But if we run it in jsdom env, there is a fetch implementation by browser, isn't it?

Reply:
no, jest is running in node so there's no `fetch` until we use a headless browser environment.

Reply:
If you specify --env=jsdom, then there IS a fetch implementation, right?

Reply:
OK, just googled it and I'm definetely wrong. Sorry about that 😂